awx_collection/plugins/modules/tower_import.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2017, John Westcott IV <john.westcott.iv@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_import
+author: "John Westcott (@john-westcott-iv)"
+version_added: "3.7"
+short_description: import resources into Ansible Tower.
+description:
+    - Import assets into Ansible Tower. See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    assets:
+      description:
+        - The assets to import.
+        - This can be the output of tower_export or loaded from a file
+      required: True
+      type: dict
+requirements:
+  - "awxkit >= 9.3.0"
+extends_documentation_fragment: awx.awx.auth
+'''
+
+EXAMPLES = '''
+- name: Import all tower assets
+  tower_import:
+    assets: "{{ export_output.assets }}"
```

Comment:
you should spell it out for people that `export_output` is from a `register:` from `tower_export`.